build.gradle.kts

```diff
@@ -12,7 +12,7 @@ plugins {
 }
 
 group = "it.krzeminski"
-version = "3.2.2-SNAPSHOT"
+version = "4.0.0-SNAPSHOT"
```

Comment:
Not sure why, changes in this file aren't visible in the diff for this PR, so adding this comment here to make it more visible.

Reply:
Ah, it's visible at the very bottom of the list of files.